### ISSUE TITLE
None of the commands work for OPed players.

### ISSUE THREAD

**Issue by EnderBro123** [DevLeoko/AdvancedBan#306]
_What version of AdvancedBan (`/AdvancedBan`) are you using?_
2.1.5-RELEASE.
 
_What kind of server do you have (`Bungeecord/single server`)?_
BungeeCord. 
 
_What server version (/version) are you using?_
1.15, using ProtocolSupport and ProtocolSupportStuff to allow 1.14-1.15.2.

_Please provide the EXACT steps required to reproduce the problem..._
1. Install on a Bungee server (Not sure if on Spigot/Paper causes same issue).
2. Try any command.
3. Inspect results.

_Any error/log post it through [pastebin.com](http://pastebin.com) and link it here._
N/A

_Add any additional information below._
Basically, none of the commands work for OPed players.

**Comment by A248** [comment]
I believe you are confused between BungeeCord versus Spigot permissions.

**Spigot Perms**
* Configured on each Spigot server, can be managed with a permissions plugin.
* Used by _Spigot plugins_.
* Includes OP which grants access to all permissions.

**Bungee Perms**
* Configured on the BungeeCord proxy.
* Used by _BungeeCord plugins_.
* There is no such thing as "OP".

**If you install AdvancedBan on BungeeCord, you will need to grant your moderators _BungeeCord_ permissions. AdvancedBan, if it is installed on BungeeCord, cannot detect Spigot permissions.**
